### DISCUSSION TITLE
How to put Python and Git in my current Codespace (Repository) ?

### DISCUSSION THREAD

**Discussion by tonyy01** [community/community#50121] [Codespaces] [answered]
### Select Topic Area

Question

### Body

My bat file needs to be run by Python and Git in its repository to execute. Is there any simple way to do this?

**Comment by cwndrws** [comment, marked as answer]
Hi 👋 The default image for Codespaces has both Git and Python in it. If you've tried using them and are having specific issues, please include the error messages that you are getting.

**Reply by tonyy01** [reply]
Thank you for your assistance! It seems that the installation is not the issue here. I'll investigate further to determine what the problem might be.
